### DISCUSSION TITLE
Project sponsorship

### DISCUSSION THREAD

**Discussion by d1b** [anthraxx/linux-hardened#62] [Ideas]
Hi @anthraxx, 
Is there a way to support your works and efforts for linux-hardened (e.g. via github sponsorship or similar) ?

**Comment by anthraxx** [comment]
That's a good idea, I will setup github sponsorship soon. It may allow me to dedicate more time on the project and resolve heavily conflicting issues faster in the future.
For now you can only reward the original work at GrapheneOS, they will be happy as well. For the current maintenance work, I try to setup the github stuff within a week so stay tuned :cat: :heart:
